src/app/config/config.go

```diff
@@ -48,19 +48,20 @@ type (
 	}
 
 	Config struct {
-		DB_DATABASE      string `toml:"DB_DATABASE"`
-		DB_HOST          string `toml:"DB_HOST"`
-		DB_USER          string `toml:"DB_USER"`
-		DB_PASS          string `toml:"DB_PASS"`
-		HOST             string `toml:"HOST"`
-		URL              string `toml:"URL"`
-		SITE_TITLE       string `toml:"SITE_TITLE"`
-		HASH_SALT        string `toml:"HASH_SALT"`
-		SLACK_WEBHOOKURL string `toml:"SLACK_WEBHOOKURL"`
-		SLACK_FACEICON   string `toml:"SLACK_FACEICON"`
-		SLACK_USERNAME   string `toml:"SLACK_USERNAME"`
-		SLACK_CNANNEL    string `toml:"SLACK_CNANNEL"`
-		FRONT_DOMEIN     string `toml:"FRONT_DOMEIN"`
+		DB_DATABASE      string   `toml:"DB_DATABASE"`
+		DB_HOST          string   `toml:"DB_HOST"`
+		DB_USER          string   `toml:"DB_USER"`
+		DB_PASS          string   `toml:"DB_PASS"`
+		HOST             string   `toml:"HOST"`
+		URL              string   `toml:"URL"`
+		SITE_TITLE       string   `toml:"SITE_TITLE"`
+		HASH_SALT        string   `toml:"HASH_SALT"`
+		SLACK_WEBHOOKURL string   `toml:"SLACK_WEBHOOKURL"`
+		SLACK_FACEICON   string   `toml:"SLACK_FACEICON"`
```

Comment:
__[golint]__
* [golint] don't use ALL_CAPS in Go names; use CamelCase

<!-- Comment by [SideCI](https://sideci.com) -->